inference-engine/src/mkldnn_plugin/nodes/mkldnn_concat_node.cpp

```diff
@@ -27,291 +27,307 @@ using namespace mkldnn;
 using namespace MKLDNNPlugin;
 using namespace InferenceEngine;
 
+
+bool MKLDNNConcatNode::isSupportedOperation(const std::shared_ptr<const ngraph::Node>& op, std::string& errorMessage) noexcept {
+    try {
+        errorMessage = std::string("CPU plugin doesn't support nGraph operation ") + op->get_type_name() + " with name '" + op->get_friendly_name() + "' ";
```

Comment:
You shouldn't put generic error prefix here since it will be added in the MKLDNNNode factory: https://github.com/dmitry-gorokhov/openvino/blob/feature/cpu_migration_on_ngraph/inference-engine/src/mkldnn_plugin/mkldnn_node.cpp#L1306
The idea is to put details into the message like: https://github.com/dmitry-gorokhov/openvino/blob/feature/cpu_migration_on_ngraph/inference-engine/src/mkldnn_plugin/nodes/gather.cpp#L33

Reply:
Fixed